Program.cs

```diff
@@ -1,2 +1,41 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine ("Hello, World!");
+﻿// Convert to Decimal to Hexadecimal and Binary Program //
+internal class Program {
+    /// <summary>Main method</summary>
+    static void Main () {
+        Console.Write ("Input a Number : ");
+        string answer = Console.ReadLine ();
+        int decimalNumber = Convert.ToInt32 (answer);
+        DecimalToHexadecimal (decimalNumber);
+        int binary = Convert.ToInt32 (answer);

```

Comment:
you have already computed the decimal number(decimalNumber), you can reuse the same variable here.